src/offset/local/tz_info/rule.rs

```diff
@@ -874,6 +874,39 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn test_from_tz_string() {
+        let tz_string = b"hhh5hxx,5,1";
+        let use_string_extensions = false;
+        assert_eq!(format!("{:?}", TransitionRule::from_tz_string(tz_string, use_string_extensions)),
```

Comment:
Using the whole `Debug` isn't precise enough, mostly, and I want to avoid having tests for code that isn't exercised by the outer API.

> Meanwhile, I assume this comment applies to all the added unit tests under `src/offset/local/tz_info` folder, is my understanding correct?

Yes.